WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryItems/CoinJoinHistoryItemViewModel.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive;
+using NBitcoin;
+using ReactiveUI;
+using WalletWasabi.Blockchain.Transactions;
+using WalletWasabi.Fluent.Extensions;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets.Home.History.HistoryItems;
+
+public class CoinJoinHistoryItemViewModel : HistoryItemViewModelBase
+{
+	public CoinJoinHistoryItemViewModel(
+		int orderIndex,
+		TransactionSummary transactionSummary,
+		WalletViewModel walletViewModel,
+		Money balance,
+		IObservable<Unit> updateTrigger)
+		: base(orderIndex, transactionSummary)
+	{
+		Label = "Coinjoin";
```

Comment:
@soosr  Why do we label the coinjoin transactions?

Reply:
It was `Privacy boost` previously. It is ok for me to remove the redundant `Coinjoin` text.

Reply:
My question is more like why do we need the label for coinjoin txs?

Reply:
Remove label text from this PR

Reply:
IMO the label in this PR was good because it is only one coinjoin.

I am just curious to know why do we even need the label for coinjoin txs.

Reply:
> I am just curious to know why do we even need the label for coinjoin txs.

We don't and I don't remember why we had that. Maybe it was an early decision and we got used to it and stick to that.

Now I don't think we need that. There is the type shield that indicates that it is a CJ transaction.
There were talks about adding the number of coinjoins into the label, but it seems that is not needed.

And also in the future, we want to show the `Recipient` in the history rather than all the entity that knows about it. `Coinjoin` text would not fit there.